src/__tests__/json.test.ts

```diff
@@ -10,8 +10,10 @@ import {
   GraphQLString,
   parse,
   GraphQLInt,
-  versionInfo
+  versionInfo,
+  GraphQLScalarType
 } from "graphql";
+import { GraphQLEmailAddress, GraphQLJSONObject } from "graphql-scalars";
```

Comment:
In GraphQL Scalars, most of the scalars have `jsonSchema` defined in the extensions;
https://github.com/Urigo/graphql-scalars/blob/master/src/scalars/EmailAddress.ts#L46
This might be helpful for you.